subprojects/integ-test/integ-test.gradle

```diff
@@ -25,8 +25,8 @@ integTestTasks.all {
     inputs.files(fileTree(userguideOutput))
 
     doFirst {
-        systemProperties['integTest.userGuideInfoDir'] = project(':docs').docbookSrc
-        systemProperties['integTest.userGuideOutputDir'] = userguideOutput.absolutePath
+        fileSystemProperty 'integTest.userGuideInfoDir', project(':docs').docbookSrc
+        fileSystemProperty 'integTest.userGuideOutputDir', userguideOutput
```

Comment:
Isn't `integTest.userGuideOutputDir` definitely an @InputDirectory here? It's used to compare against the output produced by the samples.